infer_structcol/model.py

```diff
@@ -12,6 +12,10 @@
 # define limits of validity for the MC scattering model
 min_phi = 0.35
 max_phi = 0.73
+min_radius = 70. # in nm
```

Comment:
I think that the model is valid for phi<0.35. I used it to calculate reflectances for dilute suspensions of particles with much smaller phi. The structure peak does become much more broad and flat at these values, but calculating this could still be useful for applications outside of structural color. I think we may want to consider a minimum phi close to 0. Also, for the maximum phi, maybe we should change it to .74 since that is the actual theoretical limit for crystal packing.